src/block-editor.js

```diff
@@ -6,7 +6,9 @@ const { addFilter } = wp.hooks;
 const { Fragment } = wp.element;
 const { InspectorControls } = wp.blockEditor;
 const { createHigherOrderComponent, withState } = wp.compose;
-const { ToggleControl, SelectControl, PanelBody, PanelRow } = wp.components;
+const { ToggleControl, SelectControl, PanelBody, PanelRow, TextControl } = wp.components;
```

Comment:
maybe use NumberControl rather than TextControl for your settings:
https://developer.wordpress.org/block-editor/reference-guides/components/number-control/